### DISCUSSION TITLE
Can't locate palmer plugin

### DISCUSSION THREAD

**Discussion by FrancisJMR** [jspsych/jsPsych#470] [Q&A]
Your doc refers to the palmer plugin to include the snap.svg JS library here: http://docs.jspsych.org/plugins/jspsych-palmer/

Yet loading     `<script src="static/js/snap.svg-min.js"></script>` does not help nor does the main file js/jspsych.js seem to define `jsPsych.palmer`

I'm using jsPsych 5.0.3

In general, unable to run the following
```
<html>
  <head>
    <script src="static/js/jquery.min.js"></script>
    <script src="static/js/jspsych.js"></script>
    <script src="static/plugins/jspsych-text.js"></script>
    <script src="static/plugins/jspsych-single-stim.js"></script>
    <script src="static/plugins/jspsych-visual-search-circle.js"></script>
    <script src="static/js/snap.svg-min.js"></script>
    <link rel="stylesheet" href="static/css/jspsych.css"></link>
    
  </head>
  <body>
    <div id="jspsych-target"></div>
  </body>
  <script>

  // create a stimulus using the generate_stimulus method
  var square_size = 3; // number of rows and columns
  var grid_spacing = 75; // pixels
  var circle_size = 20; // radius in pixels
  var configuration = [0,1,0,0,1,1,0,0,1,1,0,0,1,0,0,0,1,0,0,1]; // see definition above

  var stimulus = jsPsych.palmer.generate_stimulus(square_size, grid_spacing, circle_size, configuration);
```

Error:
`Cannot read property 'generate_stimulus' of undefined at test.html:24`

Thank you

**Comment by jodeleeuw** [comment]
You'll need to load the Palmer plugin, which you can find in the plugins folder. I believe that will resolve the issue.

**Comment by FrancisJMR** [comment]
Thank you! Found it on my local repo but didn't see it on the github source folder.
